### PR TITLE
UT-Alignment-12-11

### PR DESCRIPTION
**Pull request by LeeGregory12d**
## Unit test

Alignment where the vertical alignment starts after the start of the horizontal alignment
and ends before the end of the horizontal alignment.

The Axis is fudged because there is no IFC Geometry

## Covered topics

*tick one/many*

- [x] project setup
- [ ] georeferencing
- [x] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

File: UT-Alignment-12d-11/readme.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- UT-Alignment-12d-9
+
+### Content
+
+This scenario covers the additional concepts and/or IFC entities:
+
+- the vertical starts after the horizontal and stops before the end of the horizontal. 
+
+## Supporting files
```
Comment (pjanck) [inline UT-Alignment-12d-11/readme.md]:
```suggestion
### Supporting files
```

**Review by pjanck — APPROVED** [review]
LFTM

**Comment by pjanck** [comment]
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

**Review by jmirtsch — APPROVED** [review]
I have reviewed this pull request, and can successfully view the alignment curve in the Geometry Gym Rhino3d plugin. This import seems consistent with the provided description.

**Pull request merged**